source/developer/developer-setup.md

```diff
@@ -4,44 +4,46 @@ Developer Machine Setup
 ### Mac OS X ###
 
 1. Download and set up Docker Toolbox
-	1. Follow the instructions at http://docs.docker.com/installation/mac/
+	1. Follow the instructions at [`http://docs.docker.com/installation/mac/`](http://docs.docker.com/installation/mac/)
 	2. Start a new docker host  
 		`docker-machine create -d virtualbox dev`
 	2. Get the IP address of your docker host  
 		`docker-machine ip dev`
 	3. Add a line to your /etc/hosts that goes `<Docker IP> dockerhost`
 	4. Run `docker-machine env dev` and copy the export statements to your ~/.bash_profile
 2. Download Go 1.6 and Node.js using Homebrew
-	1. Download Homebrew from http://brew.sh/
+	1. Download Homebrew from [`http://brew.sh/`](http://brew.sh/)
 	2. `brew install go`
 	3. `brew install node`
 3. Set up your Go workspace
 	1. `mkdir ~/go`
-	2. Add the following to your ~/.bash_profile  
+	2. Add the following to your `~/.bash_profile`  
 		- `export GOPATH=$HOME/go`  
 		- `export PATH=$PATH:$GOPATH/bin`  
 		- `ulimit -n 8096`  
 		- If you don't increase the file handle limit you may see some weird build issues with browserify or npm.  
 	3. Reload your bash profile  
 		`source ~/.bash_profile`
-4. Download Mattermost  
-	- `cd ~/go`  
-	- `mkdir -p src/github.com/mattermost`  
-	- `cd src/github.com/mattermost`  
-	- `git clone https://github.com/mattermost/platform.git`  
-	- `cd platform`
+	4. Set GOROOT (optional) in your `~/.bash_profile`
+		- `export GOROOT=/usr/local/go/`
+4. Fork Mattermost on GitHub.com from [`https://github.com/mattermost/platform`](https://github.com/mattermost/platform), then:
+	1. `cd ~/go`  
+	2. `mkdir -p src/github.com/<username>`  
+	3. `cd src/github.com/<username>`  
+	4. `git clone https://github.com/<username>/platform.git`  
+	5. `cd platform`
 
 5. Install godep with `go get github.com/tools/godep`
-5. Run unit tests on Mattermost using `make test` to make sure the installation was successful
-6. If the tests passed, you need to create a team and an admin account. You can choose a team name, email and password:
+6. Run unit tests on Mattermost using `make test` to make sure the installation was successful
```

Comment:
You don't need to change it back, but just an FYI that Markdown automatically numbers lists like this. It's kind of annoying in a lot of cases, but it makes it easier to have long lists like this